packages/draft-js-import-element/src/stateFromElement.js

```diff
@@ -299,6 +304,13 @@ class ContentGenerator {
     let style = block.styleStack.slice(-1)[0];
     let entityKey = block.entityStack.slice(-1)[0];
     style = addStyleFromTagName(style, tagName, this.options.elementStyles);
+    let {customInlineFn} = this.options;
+    if (customInlineFn) {
+      let customInline = customInlineFn(element);
+      if (customInline != null) {
+        style = style.add(customInline);
+      }
+    }
```

Comment:
Can we add an else here for line 306. So we will only `addStyleFromTagName(...)` if customInlineFn is not specified.

Reply:
but wouldn't that mean that default styles like `<b/strong>`, `<i/em>` won't be supported if `customInlineFn` is specified?

Reply:
Oops, you're right. I guess I meant that if customInlineFn returns non-null, we should use that value instead of the `addStyleFromTagName`

Reply:
Maybe something like this:
```js
let {customInlineFn} = this.options;
let customInline = customInlineFn ? customInlineFn(element) : null;
if (customInline != null) {
  style = style.add(customInline);
} else {
  style = addStyleFromTagName(style, tagName, this.options.elementStyles);
}
```